modules/apps/foundation/contacts/contacts-web/src/main/java/com/liferay/contacts/web/notifications/ContactsCenterUserNotificationHandler.java

```diff
@@ -108,6 +102,16 @@ protected String getBody(
 				new Object[] {creatorUserName});
 		}
 
+		if ((socialRequest.getStatus() !=
+				SocialRequestConstants.STATUS_PENDING) ||
+			(socialRequest.getCreateDate() >
+				userNotificationEvent.getTimestamp())) {
```

Comment:
I guess this condition might be right, but I'm a bit confused by it. You say:

>  I added this conditional socialRequest.getCreateDate() > userNotificationEvent.getTimestamp() in 78449f5 because we can have multiple notifications referencing the same request. When a user ignore a request and the requester resend the request, it doesn't create a new request object; the existing request gets updated (see SocialRequestLocalService). If we define the notification content based only on the request's status, old notifications referencing the same request will become active, and it looks like there's duplicate notifications.

However, wouldn't socialRequest.getCreateDate() always be lower than the timestamp of any event (since they will all happen later)?